wagtailio/project_styleguide/templates/patterns/components/streamfields/looping_video_block/looping_video_block.html

```diff
@@ -0,0 +1,9 @@
+{% load wagtailcore_tags %}
+
+<div class="home-embed">
+    <div class="responsive-object" style="padding-bottom: 56.25%;">
+        <video {% if value.autoplay %}autoplay{% else %}controls{% endif %} {% if value.loop %}loop{% endif %} muted>
```

Comment:
`muted` should only be used when `autoplay` is on 🤔
one to confirm with @vossisboss

Reply:
makes sense, done in 34198484